enricher/standard/src/main/java/io/fabric8/maven/enricher/standard/ControllerViaPluginConfigurationEnricher.java

```diff
@@ -144,4 +144,18 @@ private void mergeStatefulSetSpec(StatefulSetBuilder builder, StatefulSetSpec sp
         specBuilder.endSpec();
     }
 
+    /**
+     * This method overrides the ImagePullPolicy value by the value provided in
+     * XML config.
+     *
+     * @param resourceConfig
+     * @param defaultValue
+     * @return
+     */
+    private String getImagePullPolicy(ResourceConfig resourceConfig, String defaultValue) {
```

Comment:
I would suggest moving this function to it's parent class i.e `BaseEnricher` since both `DefaultControllerEnricher` and `ControllerViaPluginConfiguration` are using this method